support-workers/src/test/scala/com/gu/support/workers/JsonFixtures.scala

```diff
@@ -405,20 +404,6 @@ object JsonFixtures {
       "0033E00001Cq8D2QAJ",
       "0013E00001AU6xcQAD",
     )
-  val salesforceContacts = {
-    SalesforceContactRecords(
-      SalesforceContactRecord(
-        "0033E00001Cq8D2QAJ",
-        "0013E00001AU6xcQAD",
-      ),
-      Some(
-        SalesforceContactRecord(
-          "0033E00001Cq8D2QAJ",
-          "0013E00001AU6xcQAD",
-        ),
-      ),
-    )
-  }
   val salesforceContactJson =
     """
```

Comment:
is salesforceContactsJson on line 430 used?

Reply:
also I guess it's not actually used as they are errors, but it occurs here:
https://github.com/guardian/support-frontend/blob/44e0443c735c223864eaaf5fd4a1bd10f54299f2/support-workers/src/test/scala/com/gu/support/workers/JsonFixtures.scala#L689
and
https://github.com/guardian/support-frontend/blob/44e0443c735c223864eaaf5fd4a1bd10f54299f2/support-workers/src/test/scala/com/gu/support/workers/JsonFixtures.scala#L727